app/src/internal/res/values/themes.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <style name="BpkTheme" parent="BpkThemeBase">
+  <style name="AppThemeBase" parent="BpkTheme">
```

Comment:
and a base theme, so we can override the fonts per variant - this is the internal one that moved to the app